backend/Dockerfile

```diff
@@ -1,10 +1,9 @@
 FROM node:alpine
 MAINTAINER alexlongerbeam
 WORKDIR /app
-RUN apk --no-cache add --virtual builds-deps build-base python
+RUN apk --no-cache add --virtual builds-deps build-base python postgresql
```

Comment:
Did you still need this? Can't remember. Seems like you run SQL straight into the postgres client programatically now

Reply:
I don't remember if I need it. I think it might be valuable to keep just in case psql is called inside the Docker container. I will remove it if you think it is unnecessary.